libs/blocks/aside/aside.js

```diff
@@ -33,6 +33,7 @@ const blockConfig = {
   },
 };
 const FORMAT_REGEX = /^format:/i;
+const mediaports = ['mobile-only', 'tablet-only', 'desktop-only'];
```

Comment:
```suggestion
const viewports = ['mobile-up', 'tablet-up', 'desktop-up'];
```
Nit: I see a few naming things that stand out to me. Just a few thoughts, take em or leave em. 
I feel like the more universal referenced name would be 'viewports' and we probably want to use explicit viewport classes. ex.`-only` isn't being honest with what it actually does. In other viewport naming conventions we've used 'mobile-only, tablet-up, and desktop-up' and general `mobile-up` wasn't needed as thats the default.